Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM nginx
+
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+RUN apt-get -qq update
+RUN apt-get -qq install git -y
+RUN apt-get -qq install nodejs -y
+RUN ln -s /usr/bin/nodejs /usr/bin/node
+RUN apt-get -qq install curl -y
+RUN curl -q https://www.npmjs.com/install.sh | sh
+COPY src /src
+COPY resources /resources
+COPY lib /lib
+COPY gulp /gulp
+COPY base.html /base.html
+COPY gulpfile.js /gulpfile.js
+COPY package.json /package.json
```

Comment:
its the dockerfile, it builds the docker image. An image has its own file-system, so i need to copy the relevant files into the image.